gradle/libs.versions.toml

```diff
@@ -0,0 +1,87 @@
+android-gradle = { module = "com.android.tools.build:gradle", version.ref = "agp" }
+twisterrob-quality = { module = "net.twisterrob.gradle:twister-quality", version.ref = "twisterrob" }
+twisterrob-android = { module = "net.twisterrob.gradle:twister-convention-plugins", version.ref = "twisterrob" }
+
+glide = { module = "com.github.bumptech.glide:glide", version.ref = "glide"}
+
+androidx-annotation = { module = "androidx.annotation:annotation", version.ref = "androidx-annotation" } 
+androidx-core-ktx = { module = "androidx.core:core-ktx", version.ref = "androidx-core" }
+androidx-appcompat = { module = "androidx.appcompat:appcompat", version.ref = "androidx-appcompat" } 
+androidx-fragment-ktx = { module = "androidx.fragment:fragment-ktx", version.ref = "androidx-fragment" } 
+androidx-test-fragment = { module = "androidx.fragment:fragment-testing", version.ref = "androidx-fragment" } 
+androidx-drawerlayout = { module = "androidx.drawerlayout:drawerlayout", version.ref = "androidx-drawerlayout" } 
+androidx-preference-ktx = { module = "androidx.preference:preference-ktx", version.ref = "androidx-preference" } 
+androidx-palette = { module = "androidx.palette:palette", version.ref = "androidx-palette" }
+androidx-palette-ktx = { module = "androidx.palette:palette-ktx", version.ref = "androidx-palette" }
+androidx-multidex = { module = "androidx.multidex:multidex", version.ref = "androidx-multidex" }
+androidx-multidex-instrumentation = { module = "androidx.multidex:multidex-instrumentation", version.ref = "androidx-multidex" }
+androidx-support = { group = "com.android.support", name = "unused" } 
+
+androidx-test-rules = { module = "androidx.test:rules", version.ref = "androidx-test" }
+androidx-test-runner = { module = "androidx.test:runner" , version.ref = "androidx-test" }
+androidx-test-espresso-core = { module = "androidx.test.espresso:espresso-core" , version.ref = "androidx-test-espresso" }
+
+junit4 = { module = "junit:junit", version.ref = "junit4" }
+junit5 = { module = "org.junit:junit-bom", version.ref = "junit5" }
+junit5-jupiter = { module = "org.junit.jupiter:junit-jupiter", version.ref = "junit5" }
+junit5-api = { module = "org.junit.jupiter:junit-jupiter-api", version.ref = "junit5" }
+junit5-android = { module = "de.mannodermaus.junit5:android-test-core", version.ref = "junit5-androidtest" }
+junit5-android-runner = { module = "de.mannodermaus.junit5:android-test-runner", version.ref = "junit5-androidtest" }
+
+mockito-core = { module = "org.mockito:mockito-core", version.ref = "mockito" }
+mockito-inline = { module = "org.mockito:mockito-inline", version.ref = "mockito" }
+mockito-android = { module = "org.mockito:mockito-android", version.ref = "mockito" }
+mockito-kotlin = { module = "org.mockito.kotlin:mockito-kotlin", version.ref = "mockito-kotlin" }
+mockito-kotlin-old = { module = "com.nhaarman.mockitokotlin2:mockito-kotlin", version = "2.2.0" }
```

Comment:
not extracted, will be removed in followup

Reply:
https://github.com/TWiStErRob/net.twisterrob.colorfilters/pull/43/commits/0d2955f4c774db93d41730f1ffef42edf78a7ef6 in #43